client/state/help/directly/actions.js

```diff
@@ -1,25 +1,33 @@
+import * as directly from 'calypso/lib/directly';
 import {
 	DIRECTLY_ASK_QUESTION,
 	DIRECTLY_INITIALIZATION_START,
 	DIRECTLY_INITIALIZATION_SUCCESS,
 	DIRECTLY_INITIALIZATION_ERROR,
 } from 'calypso/state/action-types';
-import 'calypso/state/data-layer/third-party/directly';
-
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import 'calypso/state/help/init';
 
-export function askQuestion( questionText, name, email ) {
-	return { type: DIRECTLY_ASK_QUESTION, questionText, name, email };
-}
-
-export function initialize() {
-	return { type: DIRECTLY_INITIALIZATION_START };
-}
+export const askQuestion = ( questionText, name, email ) => async ( dispatch ) => {
+	dispatch( { type: DIRECTLY_ASK_QUESTION } );
```

Comment:
Seems like we're optimistically setting the state to be in a "asked question" state, while it's not clear that `directly.askQuestion()` will be successful and if the question will actually be asked successfully. But it was like that before, so 🤷